bin/main

```diff
@@ -1,4 +1,6 @@
 #!/usr/bin/env ruby
+require './lib/logic'
+require './lib/settings'
```

Comment:
[REQUIRED] Check these paths
[OPTIONAL] Using `require_relative` for files in your directory instead of `require` is a ruby convention.